FrontEnd/FrontEnd.Web/Components/Pages/Recipes/SingleRecipe.razor

```diff
@@ -77,17 +84,39 @@ else
             </ol>
         </FluentGridItem>
     </FluentGrid>
+    
+    <FluentMenu Anchor="share-menu-button" @bind-Open="ShareMenuOpen" VerticalThreshold="170">
+        <FluentStack Orientation="Orientation.Vertical" Style="align-items: unset;">
```

Comment:
<FluentMenuItem> only supports OnClick. In that case, we would need to navigate with the current tab or inject JSRuntime to have everything working (that in turn would cause problems with popup blockers). Decided to instead workaround it by using FluentAnchor, which supports hrefs and blank target,